xml/docu_styleguide.xmlformat.xml

```diff
@@ -131,7 +131,7 @@
   </listitem>
   <listitem>
    <formalpara>
-    <title>Formatting of Computer Output/Computer Input Blocks:</title>
+    <title>Formatting of computer output/computer input blocks:</title>
```

Comment:
The normal order is input/output -- as in I/O. I think I would prefer that. Thoughts?

Reply:
@sknorr

Reply:
So...?

Reply:
Blocks often consist of just output or just input, or as it happens, a combo. But we are not referring to I/O as a term, so I don't really see a necessity to change anything here.